### ISSUE TITLE
image formatter should allow for derivative images (image styles)

### ISSUE THREAD

**Issue by jrglasgow** [contentauth/c2pa-drupal#2]
Rarely does a Drupal site display the original image to the end user. The site administrator choose image styles which defines specific ways to alter the image (scale, crop, etc...) to provide the end user with a better experience do to faster page loads because of smaller image sizes.

When choosing the display options for a image field the default Drupal core Image formatter allows the administrator to choose which image style to use. 

I would like this ability to be available using the c2pa image formatter.

I understand that  the purpose of having this module in a Drupal website would be to allow the end user to see the content credentials of the images and derivative images  would need to have those same credentials. I have been working on [this Drupal module](https://www.drupal.org/project/c2pa_sign) which will auto add signed manifests to those derivative images upon image generation.

**Comment by jrglasgow** [comment]
Ok, looking at this again to make my changes I see that the C2paImageFormatter formatter settingsForm() does inherit all of the settings from ImageFormatter, they aren't overridden. Though the settingsSummary() doesn't show it. And changing the #theme from 'image_style' to  c2pa_image_formatter does cause the image styles to not be used.

**Comment by jrglasgow** [comment]
I have created a commit in [my fork](https://github.com/contentauth/c2pa-drupal/compare/main...jrglasgow:c2pa-drupal:2?expand=1) to simplify the module by removing a unnecessary theme hook, preprocess function, and template and just added the `<div class="c2pa-wrapper">` in the render array when the C2paImageFormatter is chosen. 

I also removed the preprocess function which adds the c2pa/c2pa library to all pages loads and instead it will only be added when the C2paImageFormatter is being used (since the "c2pa-wrapper" class won't be there otherwise).

I also fixed the settingsSummary() to inherit the parent::settingsSummary() so the summary has all information.
